Packs/MicrosoftIntune/ModelingRules/MicrosoftIntune_1_3/MicrosoftIntune_1_3.xif

```diff
@@ -0,0 +1,57 @@
+[MODEL: dataset="msft_azure_raw"]
+filter _collector_name = "intune-xdr-eventhub (demistodev.onmicrosoft.com)" and category contains "DeviceComplianceOrg"
```

Comment:
demistodev.onmicrosoft.com this is demisto user name why you are using it in the filter?

Reply:
Fixed with the correct filter